evaluation.md

```diff
@@ -1,2 +1,19 @@
 # Evaluation
 
+## Diary
+
+* We implemented everything except for else's. (A problem we ran into when we tried to implement the internal DSL)
+* There are currently more seperators than we want in an example program, but we can correct for that after we deal with "else".
+* We have a working version with elses... have to use semi-colons to separate each of the states. Trying to remove those, might be harder than we think.
+* The semi-colons are gone.
+* We were using the text to test our code and now we found out that our set up was incorrect for using JFXApp and the GUIDisplay.
+* Fixed the GUIDisplay :-)
+* Added some functionality to enable change the maze
+* We ran into a strange git divergance and took a while to fix that!!
```

Comment:
What happened? Especially with git issues, if you can note what happened and how you fix it, you cold save some time in the future if it happened again.